.github/test/run-all-tests.js

```
@@ -0,0 +1,109 @@
+
+async function main() {
+  console.log(`${colors.bold}GitHub Profile Test Suite${colors.reset}`);
+  console.log('═'.repeat(50));
+  console.log(`Running tests at: ${new Date().toISOString()}`);
+  
+  // Ensure we're in the test directory
+  const currentDir = path.basename(__dirname);
+  if (currentDir !== 'test') {
+    // If not in test directory, try to change to it
+    const testDir = path.join(__dirname, __dirname.endsWith('test') ? '' : 'test');
+    if (require('fs').existsSync(testDir)) {
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)